TurboMinerVirtualTable.Unity/Assets/Scripts/Ui/MainMenu.cs

```diff
@@ -1,5 +1,8 @@
-﻿using UnityEngine;
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
 using UnityEngine.SceneManagement;
+using UnityEngine.UI;
```

Comment:
czemu te juzingi nagle tu zostaly dodane? klikaj prawym na plikach przed komitem i "remove and sort usings"

Reply:
musiały mi czmychnąć